src/Riok.Mapperly/Descriptors/WellKnownTypes.cs

```diff
@@ -41,4 +46,66 @@ internal WellKnownTypes(Compilation compilation)
 
         return typeSymbol;
     }
+
+    public ImmutableArray<AttributeData> GetAttributes(ISymbol symbol)
```

Comment:
I think this member/attribute resolution doesn't really fit to `WellKnownTypes`. I'd create a new class named `SymbolAccessor` or similar which has a private field `WellKnownTypes`.

Reply:
Should I move `GetAllMembers`, `GetAllAccessibleMappableMembers` and related dictionaries to this class?